src/js/entities-service/entities/actions.js

```diff
@@ -47,7 +47,8 @@ const _Model = BaseModel.extend({
   },
   getOwner() {
     const owner = this.get('_owner');
-    return Radio.request('entities', `${ owner.type }:model`, owner.id);
```

Comment:
@nmajor25  this is specifically the pattern needed for author to support clinician or patient for the form history.  `Store.get(author.type)` ensures the correct model and request where `Radio.request('entities', `${ owner.type }:model`` does not.